english/commit-suicide/suicide.md

```diff
@@ -1,2 +1,4 @@
 Click [here](http://www.addictdepression.com/suicide-methods/) for some ideas.
-However, you must consider [these](http://lostallhope.com/things-consider) matters first.
\ No newline at end of file
+However, you must consider [these](http://lostallhope.com/things-consider) matters first.
+
+If you would like someone to talk to about your suicidal inclinations, [here are some hotlines you can call.](https://www.reddit.com/r/SuicideWatch/wiki/hotlines)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.